src/foremast/utils/tasks.py

```diff
@@ -75,8 +76,11 @@ def _check_task(taskid):
 
     LOG.info('Checking taskid %s', taskid)
 
+    headers = copy.copy(HEADERS)
```

Comment:
Since HEADERS is only used here oddly enough, we should just remove it from consts.py. 👍